ci/scripts/clone-build_ci.sh

```diff
@@ -44,6 +44,7 @@ if [[ -d global-workflow ]]; then
   rm -Rf global-workflow
 fi
 
+export GIT_LFS_SKIP_SMUDGE=1
```

Comment:
Should not need this after the hotfix, right?

Reply:
Correct.  I'll revert this change here, in my local Jenkins clones, and in #2775 and restart CI in the latter.